Team descriptions (#25)
fishbrain/engineering-handbook: teams/teams.md

> In FishBrain, we work in purpose filled teams with their own objectives and goals. Most product teams are cross-functional teams, others can be comprised of engineers from the same domain. Below are the current teams in FishBrain.

Reviewer: Fishbrain
```suggestion
In Fishbrain, we work in purpose filled teams with their own objectives and goals. Most product teams are cross-functional teams, others can be comprised of engineers from the same domain. Below are the current teams in Fishbrain.
```

Reviewer 2: At Fishbrain.. ?

## Team Tuna

## Team Social

## Team Utility

> The team responsible for helping our 10 million users find and catch bigger fish. We enable users to plan their fishing adventures by providing an engaging map experience, ML powered spot and species prediction as well as 7 days weather forecasting. Every fish that is caught can then be added to the logbook, giving our users an overview of their own fishing activities, complete insights they can learn from.

Reviewer 3: ```suggestion
The team responsible for helping our users find and catch bigger fish. We enable users to plan their fishing adventures by providing an engaging map experience, ML powered spot and species prediction as well as 7 days weather forecasting. Every fish that is caught can then be added to the logbook, giving our users an overview of their own fishing activities, complete insights they can learn from.
```

You may have to update this often if you put a number on there (like we do with the homepage of fishbrain.com :()

Reviewer 2: Could have a banner :)

At Fishbrain, we work in purpose filled teams with their own objectives and goals. Most product teams are cross-functional teams, others can be comprised of engineers from the same domain. Below are the current teams at Fishbrain.

## Team Tuna

## Team Social

The Social team is responsible for connecting anglers on the platform, and engagement driving features. Primary caretaker of the Feed, Discover, Stories, Interactions (likes, comments, and messaging), Videos, Posts and Groups. Various functions to maintain community integrity is also a core responsibility for Social team.

## Team Utility
The team responsible for helping our users find and catch bigger fish. We enable users to plan their fishing adventures by providing an engaging map experience, ML powered spot and species prediction as well as 7 days weather forecasting. Every fish that is caught can then be added to the logbook, giving our users an overview of their own fishing activities, complete insights they can learn from.

## Team Platform

> 

Reviewer 2: ```suggestion
The platform team owns infrastructure and developer happiness. This includes but not limited to; Infrastructure configuration and monitoring, CI/CD pipelines and general tooling as well as hands on support. In addition to that we try to make sure being on-call is as hassle free as it can be, and in case of incidents we make sure there is a post mortem scheduled.
```

Reviewer 2: Added mine as a comment, crazy!

At Fishbrain, we work in purpose filled teams with their own objectives and goals. Most product teams are cross-functional teams, others can be comprised of engineers from the same domain. Below are the current teams at Fishbrain.

## Team Tuna

## Team Social

> The Social team is responsible for connecting angler on the platform, and engagement driving features. Primary caretaker of the Feed, Discover, Stories, Interacations  (likes, comments, and messaging), Moderation, Community Integrity, Videos, Post and Catch uploading. 

Author: ```suggestion
The Social team is responsible for connecting anglers on the platform, and engagement driving features. Primary caretaker of the Feed, Discover, Stories, Interactions  (likes, comments, and messaging), Videos, Posts and Groups. Moderation functions to maintain community integrity is also a core responsibility for Social team.
```

Author: I added `Groups` feature under social and try to break it down a bit. Not sure if this runs better

Reviewer: Not sure brands is going to land on our table though so I'll leave it out for now.

> Since we overlap with other teams in engagement, it can be hard to determine who go for what. The other teams focus on features, enhancements and experiments. We focus on keeping the social foundation.

Author: I am not sure if you need this paragraph actually. Feels out of place.
